python/bifrost/blocks/guppi_raw.py

```diff
@@ -59,7 +59,7 @@ def on_sequence(self, reader, sourcename):
         df_MHz = bw_MHz / nchan
         f0_MHz = cfreq_MHz - 0.5*(nchan-1)*df_MHz
         # Note: This will be negative if OBSBW is negative, which is correct
-        dt_s   = 1. / df_MHz / 1e6
+        dt_s   = np.abs(1. / df_MHz / 1e6)
```

Comment:
Hey Ben, this is one of those confusing things that doesn't really make sense. Perhaps more flexible compromise for the end user while being 'correct' is adding a `reverse_scale` view to flip the -ive sign? 

The `fine_time` axis *has* to run in the same direction as the `time` axis IMO. Negative time isn't really physical, it arises due to degeneracy between the negative frequency introduced by mixing with an LO and taking the upper sideband, i.e. `f_IF = f_LO - f_RF`. (do we agree on this?)

I agree that the FFT preserves the sign of the axis scale, and that `coarse channel` can be considered -ive scale.  I think the key is *can be considered*, and that if it is considered, you have to consider *both* `time` and `fine_time` to be -ive. But time definitely flows forward ;)

Another way of thinking about it that I haven't squared away: if we looked at a single pulse from a pulsar (squared each coarse channel and did an imshow), it would be flipped in frequency but not time?